### PR TITLE
Play and objectives icon addition

### PR DESCRIPTION
**Pull request by fullcolorcoder**
Resolves: [MWPW-121791](https://jira.corp.adobe.com/browse/MWPW-121791)

**Test URLs:**
[Before](https://main--milo--adobecom.hlx.page/drafts/bradjohn/document1?martech=off)
[After](https://customer-story-icons--milo--adobecom.hlx.page/drafts/bradjohn/document1?martech=off)

Bacom:
https://bradjohn-customer-story-svg--bacom--adobecom.hlx.page/drafts/bradjohn/document?milolibs=customer-story-icons&martech=off

### CODE REVIEW

File: libs/img/icons/icons.svg [hunk 1]
```diff
@@ -12,5 +12,21 @@
   <symbol id="play-circle" viewBox="0 0 18 18">
     <path fill="currentcolor" d="M9,1a8,8,0,1,0,8,8A8,8,0,0,0,9,1Zm4.2685,8.43L7.255,12.93A.50009.50009,0,0,1,7,13H6.5a.5.5,0,0,1-.5-.5v-7A.5.5,0,0,1,6.5,5H7a.50009.50009,0,0,1,.255.07l6.0135,3.5a.5.5,0,0,1,0,.86Z" />
   </symbol>
+  <symbol id="icon-results" viewBox="0 0 18 18">
```
Comment (auniverseaway) [inline libs/img/icons/icons.svg:15]: I think the word icon is redundant.

**Review by auniverseaway — CHANGES_REQUESTED** [review]
Small change. :-)

File: libs/img/icons/icons.svg [hunk 2]
```diff
@@ -12,5 +12,21 @@
...
+    <rect id="Canvas" fill="currentcolor" opacity="0"/>
+    <path fill="currentcolor" d="M13,1.5V15h3V1.5a.5.5,0,0,0-.5-.5h-2A.5.5,0,0,0,13,1.5Z" />
+    <path fill="currentcolor" d="M9,6.5V15h3V6.5a.5.5,0,0,0-.5-.5h-2A.5.5,0,0,0,9,6.5Z" />
+    <path fill="currentcolor" d="M5,10.5V15H8V10.5a.5.5,0,0,0-.5-.5h-2A.5.5,0,0,0,5,10.5Z" />
+    <path fill="currentcolor" d="M1,12.5V15H4V12.5a.5.5,0,0,0-.5-.5h-2A.5.5,0,0,0,1,12.5Z" />
+    <rect fill="currentcolor" rx="0.25" y="16" />
+  </symbol>
 
+  <symbol id="icon-objectives" viewBox="0 0 18 18">
```
Comment (ryanmparrish) [inline libs/img/icons/icons.svg:24]: Same w/ this one. `icon-` can be omitted. from the id.

**Review by ryanmparrish — APPROVED** [review]

**Review by chrischrischris — APPROVED** [review]

**Comment by ryanmparrish** [comment]
Seem like this is GTG. Verified!

**Pull request merged**